tests/installation/product_selection/install_SLES.pm

```diff
@@ -5,11 +5,15 @@
 # Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
 
 use base 'y2_installbase';
+use testapi 'get_var';
 use strict;
 use warnings;
 
 sub run {
-    $testapi::distri->get_product_selection()->install_product('SLES');
+    my $timeout = 120 * get_var('TIMEOUT_SCALE', 1);
+    my $product_selection = $testapi::distri->get_product_selection();
+    $product_selection->get_product_selection_page({timeout => $timeout, interval => 10, message => 'Product Selection page is not displayed'});
```

Comment:
how does it look the error here? we normally create a specific function for waiting, not reusing the existing one, there is a die message and and message you passed here, one of them might not be displayed, right?

Reply:
I've added another function `wait_product_selection_page` and avoid touching the Base.pm for regression.  and updated all VRs.  please take a look again. Thanks.